WalletWasabi.Fluent/Behaviors/ExecuteCommandOnDoubleTapped.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Reactive.Disposables;
+using System.Windows.Input;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class ExecuteCommandOnDoubleTapped : DisposingBehavior<Control>
+{
+	public static readonly StyledProperty<ICommand?> CommandProperty =
+		AvaloniaProperty.Register<ExecuteCommandOnDoubleTapped, ICommand?>(nameof(Command));
+
+	public ICommand? Command
+	{
+		get => GetValue(CommandProperty);
+		set => SetValue(CommandProperty, value);
+	}
+
+	protected override void OnAttached(CompositeDisposable disposables)
+	{
+		if (AssociatedObject is { })
```

Comment:
Is this null check necessary? `Equals()` handles null.

Reply:
Ah yes in that case it's not

Reply:
Pushed fix